### DISCUSSION TITLE
Moose-language-support errors after version 1.0

### DISCUSSION THREAD

**Discussion by stonebeta** [idaholab/moose#28677] [Q&A Tools] [answered]
I've encountered an issue after upgrading moose-language-support to version 1.0 and above.
![image](https://github.com/user-attachments/assets/dc917d56-5e4b-4124-88c4-1594a53d1429)

Do you have any idea what might be causing this issue. How I could resolve it?

**Comment by GiudGiud** [comment, marked as answer]
Hello

It looks like the version of the language server is requesting content that moose is not producing.
You can try updating moose, or you can to downgrade the version of the moose-language-support extension.

Guillaume
